### PR TITLE
[toolbars-modals/ui-components] collapse components

### PR DESCRIPTION
**Pull request by tamrat-bay**

### CODE REVIEW

File: packages/toolbars-modals/web/src/link/LinkComponents/LinkPanel.jsx [hunk 1]
```diff
@@ -134,6 +153,89 @@ class LinkPanel extends Component {
...
+            </Popover>
+          </>
+        )}
+      </PopoverWrapper>
+    );
+
+    const checkboxesData = [
+      {
+        showCheckbox: showNoReferrerCheckbox,
+        label: t('LinkPanel_Noreferrer_Checkbox_Label'),
```
Comment (tamrat-bay) [inline packages/toolbars-modals/web/src/link/LinkComponents/LinkPanel.jsx:183]: label keys - change on experiment toggle

File: packages/toolbars-modals/web/src/link/LinkComponents/LinkPanel.jsx [hunk 2]
```diff
@@ -134,6 +153,89 @@ class LinkPanel extends Component {
...
+        checked: noreferrer,
+        dataHook: 'linkPanelNoreferrerCheckbox',
+        onChange: this.handleNoReferrerChange,
+        tooltipTextKey: t('LinkPanel_Noreferrer_Tooltip'),
+      },
+      {
+        showCheckbox: showNoFollowCheckbox,
+        label: t('LinkPanel_Nofollow_Checkbox_Label'),
```
Comment (tamrat-bay) [inline packages/toolbars-modals/web/src/link/LinkComponents/LinkPanel.jsx:191]: label keys - change on experiment toggle

File: packages/toolbars-modals/web/src/link/LinkComponents/LinkPanel.jsx [hunk 3]
```diff
@@ -134,6 +153,89 @@ class LinkPanel extends Component {
 
     const { targetBlank, nofollow, sponsored, noreferrer } = linkValues;
 
+    const LearnMoreLink = () => (
+      <a
+        className={styles.linkPanel_tooltip_link}
+        target="_blank"
+        href={'https://support.wix.com/en/article/adding-link-rel-attributes'}
+        rel="noreferrer noopener"
```
Comment (tamrat-bay) [inline packages/toolbars-modals/web/src/link/LinkComponents/LinkPanel.jsx:161]: noreferrer

File: packages/editor-common/web/src/Components/LinkComponents/LinkPanel.jsx [hunk 4]
```diff
@@ -150,37 +150,43 @@ class LinkPanel extends Component {
...
-                tooltipTextKey={'LinkPanel_Nofollow_Checkbox_Tooltip'}
-                t={t}
-                isMobile={isMobile}
-              />
+              <div className={styles.linkPanel_checkbox}>
+                <Checkbox
+                  label={t('LinkPanel_Nofollow_Checkbox')}
+                  theme={theme}
+                  checked={nofollow}
+                  dataHook="linkPanelRelCheckbox"
```
Comment (danarish4) [inline packages/editor-common/web/src/Components/LinkComponents/LinkPanel.jsx:169]:
```suggestion
                  dataHook="linkPanelNofollowCheckbox"
```

File: packages/toolbars-modals/web/src/link/LinkComponents/LinkPanel.jsx [hunk 5]
```diff
@@ -134,6 +153,97 @@ class LinkPanel extends Component {
...
+        {({ showTooltip, hideTooltip, active, parent }) => (
+          <>
+            <InfoIcon onMouseEnter={showTooltip} onMouseLeave={hideTooltip} />
+            <Popover active={active} parent={parent}>
+              {content} <LearnMoreLink />
+            </Popover>
+          </>
+        )}
+      </PopoverWrapper>
+    );
```
Comment (danarish4) [inline packages/toolbars-modals/web/src/link/LinkComponents/LinkPanel.jsx:178]: split to another file `LearnMoreLink`

File: packages/toolbars-modals/web/src/link/LinkComponents/LinkPanel.jsx [hunk 6]
```diff
@@ -134,6 +153,97 @@ class LinkPanel extends Component {
...
+                <div className={classNames(styles.linkPanel_advancedSettings_mobile_text_info)}>
+                  {t(checkbox.tooltipTextKey)} <LearnMoreLink />
+                </div>
+              ) : this.useAdvancedSettings ? (
+                <InfoIconWithTooltip content={t(checkbox.tooltipTextKey)} />
+              ) : null}
+            </div>
+          )
+      );
+    };
```
Comment (danarish4) [inline packages/toolbars-modals/web/src/link/LinkComponents/LinkPanel.jsx:245]: create `AdvancedSettings` component

**Review by danarish4 — CHANGES_REQUESTED** [review]
After merging this we need to remember to add BI & e2e

**Pull request merged**